CodeWritingCow/cnse
# API Assignment 01: Implement Voter API

## Description

This PR implements a basic voter API for Assignment 1.

## How Has This Been Tested?

1. Open voter-api root folder in two terminal windows
2. Run the command `make run` in the first terminal
3. Run the command `go test ./tests -v` in the second terminal
4. Verify that all tests pass
